imports/pages/groups/profile/index.js

```diff
@@ -19,6 +19,24 @@ import { modal } from "../../../data/store";
 import Layout from "./Layout";
 import Join from "./Join";
 
+const PHONE_QUERY = gql`
+{
+  currentPerson {
+    phoneNumbers {
+      number
+      rawNumber
+    }
+  }
+}`;
+
+export const JoinWithPhones = graphql(PHONE_QUERY, {
+  name: "phoneNumbers",
```

Comment:
@JakeDawkins like this:

```js
const PHONE_QUERY = gql`
query PhoneNumbers {
 ...
}`;
```

That way @richarddubay can use http://dev.apollodata.com/react/cache-updates.html#updateQueries